tests/create-job.posthog.test.ts

```
@@ -0,0 +1,340 @@
+}
+
+// ---------------------------------------------------------------------------
+// Tests
+// ---------------------------------------------------------------------------
+
+describe("CreateJob PostHog Metering", () => {
+  test("VAL-CJ-PH-001: successful job generation emits builder.template.generated event", async () => {
+    installHappyPathMocks();
+    const { executeCreateJob } = await import(
+      "../src/api/v2/agent-templates/services/job-executor"
+    );
```

ESLint

[error] Replace `·await·import(⏎······"../src/api/v2/agent-templates/services/job-executor"⏎····` with `⏎······await·import("../src/api/v2/agent-templates/services/job-executor"`

(prettier/prettier)